### ISSUE TITLE
Projections availability card updates

### ISSUE THREAD

**Issue by bgsandan** [micronutrientsupport/micronutrient-support-tool#494]
![](https://user-images.githubusercontent.com/39411049/120444386-7b30f100-c37f-11eb-9f54-3ab32059547f.png)

*    [x]  Update title to "Projected \<Micronutrient> availability to 2050
*   [x] Update y access to "\<Micronutrient> availability in mg/capita/day
*   [x] Add a horizontal line to indicate the threshold value (as displayed fetched from the 'summary' endpoint in the projection summary card
*   [x] Hover values to round to 3.s.f
*   [x] Make series fill solid to match legend colours

![](https://user-images.githubusercontent.com/39411049/120444928-07431880-c380-11eb-8c08-e2d86527507a.png)

*    [x]  Only display the columns relating to the chosen micronutrient (e.g. for Calcium - Country, Year, Scenario, Ca, Ca Diff)
*    [x]  Update title header to read "Projected \<Micronutrient> availability to 2050" rather than "Multinutrient availability"
*    [x]  Update description text to read "The data shows the projected \<Micronutrient> availability per capita per day
*    [x]  Update column title for chosen MN to e.g. "Ca availability"
*    [x]  Update 'Diff' column title to "Difference from previous year"
*    [ ]  Add a % sign to values in the diff column

**Comment by Fuhji** [comment]
The reason why the fill and legend were not matching is due to the fill transparency, so overlapping data affects the colour displayed. 

Removing the fill solves this problem.

@pautva Can you suggest a palette of 3 colours for line display which are not purple/cyan, which would compliment the UI? Ive placeholder RGB for now

![image](https://user-images.githubusercontent.com/4210022/120609018-b566c500-c449-11eb-89dc-d1a4067bc60c.png)

**Comment by Fuhji** [comment]
@bgsandan Which attribute is needed for the horizontal threshold returned from the summary?

Options: 
- `referenceVal`
- `target`

![image](https://user-images.githubusercontent.com/4210022/120609726-62414200-c44a-11eb-966a-521ada2e6c01.png)

**Comment by bgsandan** [comment]
That will be `target` :+1:

**Comment by pautva** [comment]
@Fuhji, some colour suggestions for the line chart:

* A lighter brand colour **(#9B51E0)** for **SSP2** - middle of the road scenario
* A brighter danger colour **(#FF3E7A)** for **SSP3**  - negative pathway
* Happy green **(#6FCF97)** for **SSP1**  - sustainable development

It could look like this:

![Graph-colours-maps](https://user-images.githubusercontent.com/72566636/120819444-7de04180-c54b-11eb-9e25-f1c935735202.png)

**Comment by Fuhji** [comment]
> @Fuhji, some colour suggestions for the line chart:
> 
> * A lighter brand colour **(#9B51E0)** for **SSP2** - middle of the road scenario
> * A brighter danger colour **(#FF3E7A)** for **SSP3**  - negative pathway
> * Happy green **(#6FCF97)** for **SSP1**  - sustainable development
> 
> It could look like this:
> 
> ![Graph-colours-maps](https://user-images.githubusercontent.com/72566636/120819444-7de04180-c54b-11eb-9e25-f1c935735202.png)

@pautva colours look great, although i dont think we can assign a colour based on pathway positive/negative. In which case any issues with just assigning colours to SSP as shown above?

**Comment by pautva** [comment]
Yes, I think we can assign colours based just on SSP. From what I understand, SSP1 will always be "Sustainable development", SSP2 - "Middle-of-the-road" and SSP3 "Negative future pathway". So the colours for them shouldn't change. 

![image](https://user-images.githubusercontent.com/72566636/121153987-70260700-c83e-11eb-979c-aab15d77dc2b.png)
